src/v2/api/index.md

```diff
@@ -674,14 +674,15 @@ type: api
       },
       // string method name
       b: 'someMethod',
-      // deep watcher
+      // the callback will be called whenever any of the watched object properties change regardless of their nested depth
       c: {
         handler: function (val, oldVal) { /* ... */ },
         deep: true
       },
       // the callback will be called immediately after the start of the observation
       d: {
-        handler: function (val, oldVal) { /* ... */ },
+        // use a method name, instead of an anonymous function
```

Comment:
This looks a bit misleading that people may assume using method name here is required. Maybe we can use something like:

```suggestion
        // string method name also works here
```

Reply:
Or "a method name also works here" because "name" already implies a string.